### PR TITLE
Updating copyright headers to match current best practices

### LINKED ISSUE #1: Add license header to top of all code files
See https://opensource.google/docs/releasing/licenses/

We can either use;
```
Copyright (C) 2020  The SymbiFlow Authors.

Use of this source code is governed by a ISC-style
license that can be found in the LICENSE file or at
https://opensource.org/licenses/ISC
```

Or put the full license at the top;
```
Copyright (C) 2020  The SymbiFlow Authors.

Permission to use, copy, modify, and/or distribute this software for any
purpose with or without fee is hereby granted, provided that the above
copyright notice and this permission notice appear in all copies.

THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
```

### PR DESCRIPTION
**Pull request by kgugala**
fixes #1

### CODE REVIEW

**Review by mithro — CHANGES_REQUESTED** [review]
Please go through the checklist at https://github.com/SymbiFlow/ideas/issues/47 and confirm everything there has been done, then re-request review!

File: python/symbiflow_test.py [hunk 1]
```diff
@@ -1,4 +1,12 @@
 #!/usr/bin/env python3
+# Copyright (C) 2020  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC\
```
Comment (mithro) [inline python/symbiflow_test.py]: Random extra `\`

File: python/symbiflow_test.py [hunk 2]
```diff
@@ -1,4 +1,12 @@
 #!/usr/bin/env python3
```
Comment (mithro) [inline python/symbiflow_test.py:1]:
```
#
```

File: python/symbiflow_test.py [hunk 3]
```diff
@@ -1,4 +1,12 @@
...
+# Copyright (C) 2020  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC\
+#
+# SPDX-License-Identifier: ISC
+#
```
Comment (daniellimws) [inline python/symbiflow_test.py]: Extra newline

Comment (tmichalak) [reply python/symbiflow_test.py]: Fixed

File: python/symbiflow_test.py (hunk 1 above)
Comment (tmichalak) [reply python/symbiflow_test.py]: Fixed

File: python/symbiflow_test.py (hunk 2 above)
Comment (tmichalak) [reply python/symbiflow_test.py:1]: Made the license header look exactly as we have in prjxray

**Pull request merged**